core/src/main/java/io/questdb/cairo/pool/AbstractMultiTenantPool.java

```diff
@@ -159,6 +159,20 @@ public boolean lock(TableToken tableToken) {
         return true;
     }
 
+    public void notifyDropped(TableToken token) {
+        Entry<T> e = entries.get(token.getDirName());
+        long thread = Thread.currentThread().getId();
+        while (e != null) {
+            for (int i = 0; i < ENTRY_SIZE; i++) {
+                if (Unsafe.cas(e.allocations, i, UNALLOCATED, thread)) {
+                    closeTenant(thread, e, i, PoolListener.EV_EXPIRE, PoolConstants.CR_DROPPED);
```

Comment:
nit: may be we can use dedicated event here? EV_DROPPED

Reply:
done